NOTICE.txt

```diff
@@ -0,0 +1,9 @@
+Apache Arrow Website
+Copyright 2017-2022 The Apache Software Foundation
+
+This product includes software developed at
+The Apache Software Foundation (http://www.apache.org/).
+
+This product includes files generated by Jekyll (MIT)
```

Comment:
I think that the source doesn't include files generated by Jekyll. The processed files at https://github.com/apache/arrow-site/tree/asf-site include files generated by Jekyll.

Could you improve this sentence?

Reply:
Excellent point! I've updated the wording to clarify that the generated Jekyll files are included in the deployed version of the website (rather than in the source tree).